scripts/linux/pkg/deb/build-deb_ubuntu-bionic_amd64.ps1

```diff
@@ -1,4 +1,7 @@
-$env:OUTPUT_DIR = "$PSScriptRoot/dist"
```

Comment:
You do not use these scripts (build-deb_*) do you?

Reply:
No, actually, I don't. Shall I remove them?